sql/catalog.go

```diff
@@ -44,8 +42,8 @@ type (
 func NewCatalog() *Catalog {
 	return &Catalog{
 		FunctionRegistry: NewFunctionRegistry(),
-		IndexRegistry:    NewIndexRegistry(),
-		ViewRegistry:     NewViewRegistry(),
+		//IndexRegistry:    NewIndexRegistry(),
+		//ViewRegistry:     NewViewRegistry(),
```

Comment:
You now access the IndexRegistry and ViewRegistry through the sql.Context object